docs/SERVER-API.md

```diff
@@ -0,0 +1,214 @@
+This document details the API offered by the server to clients. Each method description contains:
+- The method's name
+- The method's purpose and intended effect on the server and/or the appearance of the screen rendered by zellij.
+- The method's parameters
+- The response that the client can expect to receive, including any possible error messages. N.B. Response does not detail side-effects of the request, in particular changes in what's rendered on screen.
+
+The document does not describe implementation details on the server side and is intended for use by those implementing new zellij clients.
+
+
+## Basic controls
+
+### Write input
+
+**Purpose:** Writes a set of bytes to the terminal, not to be interpeted by zellij.
```

Comment:
I'd change this (just to be explicit) to: "Writes a set of bytes to the focused pane"